zulipterminal/ui_tools/views.py

```diff
@@ -1376,6 +1385,7 @@ def __init__(self, controller: Any, stream_id: int) -> None:
                         f"Press {email_keys} to copy Stream email address",
                     ),
                     ("History of Stream", f"{availability_of_history}"),
+                    ("Posting Policy", f"{stream_policy}"),
```

Comment:
We're adding new functionality here, so it is not a refactor commit. In any case we should mention the file being updated, as in the zulip-terminal code style section.